torch/distributed/_tensor/experimental/attention.py

```diff
@@ -392,6 +467,7 @@ def _templated_ring_attention_backward(
             pointer += value.numel()
 
         if i != size - 1:
+            # Send the kv to the next rank.
```

Comment:
so is world_size for CP typically 2, and that matches up to 2 chunks and 2 rotation steps? or am i confusing stripe-chunks with algorithm steps